packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -35,6 +37,17 @@ interface LanguageConfig {
   };
 }
 
+type Permission = 'pull' | 'push' | 'admin' | 'maintain' | 'triage';
```

Comment:
I'm fine having this here, but I also know this is expressed somewhere in OctoKit already :)

Reply:
Looks like octokit does not publish a type or enum for these constants